src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/VariantAnnotationArgumentCollection.java

```diff
@@ -54,4 +54,10 @@ public VariantAnnotationArgumentCollection(List<String> defaultGroups, List<Stri
     @Argument(fullName= StandardArgumentDefinitions.ANNOTATION_GROUP_LONG_NAME, shortName=StandardArgumentDefinitions.ANNOTATION_GROUP_SHORT_NAME, doc="One or more groups of annotations to apply to variant calls", optional=true)
     public List<String> annotationGroupsToUse = new ArrayList<>();
 
+    /**
+     * Hook allowing for the user to remove default annotations from the tool
+     */
+    @Argument(fullName= StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS, shortName=StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS, doc="Disable all tool default annotations", optional=true)
```

Comment:
As I suggested when the `VariantAnnotationArgumentCollection` was introduced, I would like to have this class as an abstract implementation, because if not some arguments cannot be hidden (e.g., I do not want to expose annotation groups, but only annotations to include/exclude).

This new argument is also an example of not exposing the disabling of all the annotations... For downstream projects, that will be really nice to be able to expose only some arguments...

Reply:
Right- we did this for read filters, so unless there is a compelling reason otherwise, we should do that here too (and, this PR already includes a constructor that takes a VariantAnnotationArgumentCollection, which isn't useful unless we implement an abstract base class for args).

Reply:
Added